chart/datadog-operator/templates/clusterrole.yaml

```diff
@@ -57,4 +72,166 @@ rules:
   - "list"
   - "get"
   - "watch"
+- apiGroups:
+  - autoscaling
+  resources:
+  - horizontalpodautoscalers
+  verbs:
+  - list
+  - watch
+## To fix: should be role not a clusterrole
+- apiGroups:
+  - ""
+  resourceNames:
+  - datadogtoken
+  - datadog-leader-election
+  - datadog-custom-metrics
+  - extension-apiserver-authentication
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - create
+  - get
+  - update
+# Openshift
+- apiGroups:
+  - quota.openshift.io
+  resources:
+  - clusterresourcequotas
+  verbs:
+  - list
+  - get
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+  resourceNames:
+  - restricted
+# compliance RBAC
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  - namespaces
+  verbs:
+  - list
+- apiGroups:
+  - policy
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
+  - roles.rbac.authorization.k8s.io
+  - authorization.k8s.io
+  resources:
+  - clusterroles
+  - clusterrolebindings
+  - rolebindings
+  - roles
+  verbs:
+  - "*"
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - networkpolicies
+  verbs:
+  - list
+# Admission webhook
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - mutatingwebhookconfigurations
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
```

Comment:
why do we define these twice? we have them defined above

Reply:
Also the "secrets" here look wrong.

Reply:
wrong cut/past

Reply:
same thing in deploy/clusterrole